src/provider/WalletLinkProvider.ts

```diff
@@ -269,6 +269,15 @@ export class WalletLinkProvider
     }
 
     const relay = await this.initializeRelay()
+      const isWhitelistedNetwork = !relay.supportsUnauthedAddEthereumChain(chainId.toString())
```

Comment:
Sorry I think this is more confusing :D .  The reason is that it's not really a boolean for is whitelisted network 
`supportsUnauthedAddEthereumChain` = `true` if `white listed network || standalone extension`

I suggest just doing this:
WalletLinkRelay:
```
inlineAddEthereumChain(chain: string): boolean {
  return this.ui.inlineAddEthereumChain(chain)
}
```

```
const isWhiteListedNetworkOrStandalone = relay.inlineAddEthereumChain(chainId)
```